gen_png.py

```diff
@@ -145,17 +159,24 @@ def main():
             elif 'D' == split_file[0]:
                 deleted.append(split_file[1].removeprefix('svg/').removesuffix('.svg'))
             elif re.match(r"R[0-9]+", split_file[0]):
-                deleted.append(split_file[1].removeprefix('svg/').removesuffix('.svg'))
-                modifications.append(split_file[2])
+                renamed_old.append(split_file[1].removeprefix('svg/').removesuffix('.svg'))
+                renamed_new.append(split_file[2])
             else:
                 print(f'Unrecognized git action: {split_file}')
                 sys.exit(1)
 
 #        print('A:' + str(additions))
 #        print('M:' + str(modifications))
 #        print('D:' + str(deleted))
-#        print('R[0-9]+:' + str(renamed))
-
+#        print(renamed_old)
+#        print(renamed_new)
+
+        print('\nRenamed:')
```

Comment:
Wouldn't it be better to keep consistency with the `create_tag` function, and do these in order of Additions, modifications, renamed, deletions?